plugins/woocommerce/tests/e2e-pw/tests/merchant/product-delete.spec.js

```diff
@@ -0,0 +1,121 @@
+const {test: baseTest, expect} = require('@playwright/test');
+const wcApi = require('@woocommerce/woocommerce-rest-api').default;
+
+baseTest.describe('Products > Delete Product', () => {
+	baseTest.use({storageState: process.env.ADMINSTATE});
+
+	const apiFixture = baseTest.extend({
+		api: async ({baseURL}, use) => {
+			const api = new wcApi({
+				url: baseURL,
+				consumerKey: process.env.CONSUMER_KEY,
+				consumerSecret: process.env.CONSUMER_SECRET,
+				version: 'wc/v3',
+				axiosConfig: {
+					// allow 404s, so we can check if the product was deleted without try/catch
+					validateStatus: function (status) {
+						return status >= 200 && status < 300 || status === 404;
+					},
+				},
+			});
+
+			await use(api);
+		},
+	});
+
+	const test = apiFixture.extend({
+		productId: async ({baseURL, api}, use) => {
```

Comment:
I think `baseURL` here is stated but never called. Can be removed.

Reply:
Indeed. Removed with https://github.com/woocommerce/woocommerce/pull/43518/commits/b36fff4b3c5e9649565f624702335cb1bf121e72